clusters/app.ci/quayio-pull-through-cache/quayio-pull-through-cache-us-east-1.yaml

```diff
@@ -68,11 +113,20 @@ items:
               secretKeyRef:
                 name: quayio-pull-through-cache-credentials
                 key: AWS_SECRET_ACCESS_KEY
-          - name: REGISTRY_STORAGE_S3_BUCKET
```

Comment:
unless I miss something I think I still prefer common, shared config map + variables per bucket.

Reply:
yeah, but for every aws deployment the file would be the same.